src/main/java/com/google/api/codegen/viewmodel/RetryPairDefinitionView.java

```diff
@@ -0,0 +1,64 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.viewmodel;
+
+import com.google.api.gax.core.RetrySettings;
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableSet;
+
+@AutoValue
+public abstract class RetryPairDefinitionView {
+
+  public abstract Name name();
+
+  public abstract ImmutableSet<String> retryCodes();
```

Comment:
Hmm I don't know that that is common knowledge. In any case, the primary "idea" of a set is to have no repeated elements; the primary "idea" of a list is its ordering. I would prefer the type to reflect that purpose, and in the rendering scenario, the ordering is more important. The no repeated elements aspect is more algorithmic and is a concern of the transformer.